client/src/components/video.tsx

```diff
@@ -63,11 +63,11 @@ const Video = (args: VideoParams) => {
       : "mobile";
   const video = {
     src: videoState.mentor
-      ? videoState.isIdle
-        ? videoState.idleVideoId
-          ? videoUrl(videoState.mentor, videoState.idleVideoId, format)
-          : idleUrl(videoState.mentor, format)
-        : videoUrl(videoState.mentor, videoState.answerId, format)
+      ? videoUrl(
```

Comment:
do we want/need this change or is the fix really all about the `isPlaying` thing above?